pulsar/json_schemas/channel.json

```diff
@@ -0,0 +1,73 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://asyncapi.com/bindings/pulsar/channel.json",
+    "title": "Channel Schema",
+    "description": "This object contains information about the channel representation in Pulsar.",
+    "type": "object",
+    "additionalProperties": false,
+    "patternProperties": {
+      "^x-[\\w\\d\\.\\-\\_]+$": {
+        "$ref": "https://raw.githubusercontent.com/asyncapi/spec-json-schemas/v2.14.0/schemas/2.4.0.json#/definitions/specificationExtension"
+      }
+    },
+    "properties": {
+      "namespace": {
+        "type": "string",
+        "description": "The namespace, the channel is associated with. If omitted, 'public' MUST be assumed."
+      },
+      "persistence": {
+        "type": "string",
+        "enum": [
+          "persistent",
+          "non-persistent"
+        ],
+        "description": "persistence of the topic in Pulsar."
+      },
+      "compaction": {
+        "type": "integer",
+        "minimum": 0,
+        "description": "Topic compaction threshold given in MB"
+      },
+      "retention": {
+        "type": "object",
+        "additionalProperties": false,
+        "properties": {
+          "time": {
+            "type": "integer",
+            "minimum": 0,
+            "description": "Time given in Minutes. `0` = Disable message retention (by default)"
+          },
+          "size": {
+            "type": "integer",
+            "minimum": 0,
+            "description": "Size given in MegaBytes. `0` = Disable message retention (by default)"
+          }
+        }
+      },
+      "deduplication": {
+        "type": "boolean",
+        "description": "Whether deduplication of events is enabled or not."
+      },
```

Comment:
Do we need to define the other pulsar components here?
`partition`
`ttl`,
`dead-letter-queue`?

Reply:
I have added TTL, good eye. Completely neglected it.